src/QueryClientProvider.tsx

```diff
@@ -9,11 +9,13 @@ import { useStore } from "./Store";
 import { createAsyncStoragePersister } from "@tanstack/query-async-storage-persister";
 import { ReactQueryDevtools } from "@tanstack/react-query-devtools";
 
+const GARBAGE_COLLECTION_TIME = 1000 * 60 * 60 * 24 * 7; // 1 week
```

Comment:
I was going to suggest using a word like "interval" or "duration" instead of "time," given that the value is not a timestamp (but is, instead, a temporal "distance" or "width").

```suggestion
const GARBAGE_COLLECTION_INTERVAL = 1000 * 60 * 60 * 24 * 7; // 1 week
```

Then, I noticed the names of the properties of the `defaultOptions.queries` object—they end with "`...Time`". In that case, although I still prefer "`..._INTERVAL`" here, I see a benefit of keeping this constant's name ending with `..._TIME`".

Reply:
Point taken, but I think I'll rather stay consistent with TanStack Query's `gcTime` option and all their docs.